css/properties/page.json

```
@@ -25,7 +25,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "≤13.1"
+              "version_added": "≤4"
```

I have tested locally and can confirm that it's supported since at least Safari 1.1, so probably since Safari 1:

```suggestion
              "version_added": "1"
```